neutone_sdk/audio.py

```diff
@@ -19,6 +21,32 @@
 log = logging.getLogger(__name__)
 
 
+def write_mp3(filename: str, y: np.array, sr: int, quality: float = 0):
+    """
+    We're using this instead of sf.write in order to change the bitrate,
+    where quality goes from 0 (high) to 1 (low).
+
+    Like with sf.write y should be (num_samples, num_channels).
```

Comment:
since we are using the (num_samples, num_channels) convention here which is the opposite of what's done in PyTorch and therefore prob our users, can we add an assert like `assert y.shape[0] > y.shape[1], "Expecting  audio to have a shape of (num_samples, num_channels), try swapping the dimensions"` to allow for quick debugging of the error

Reply:
Done!